KeywordSyntax.md

```diff
@@ -32,3 +32,49 @@ and Guile, Kawa will support it with an option.
 
 If we adopt the `#:foo` style, Chicken, Guile, S7, and to some extent Racket and Kawa
 will work out of the box.
+
+## Tokens consisting only of colons
+
+How is the datum `:` (one colon) read?
+
+* As a **keyword** with the name `""` in Common Lisp, Gauche,
+  Sagittarius, STklos.
+
+* As a **symbol** with the name `":"` (one colon) in s7, Bigloo,
+  Chicken, Gambit, s7.
+
+How is the datum `::` (two colons) read?
+
+* As a **keyword** with the name `""` in Common Lisp.
```

Comment:
Please don't say "Common Lisp" unless it's in the spec (which says ::foo is undefined) or you have tested all the Common Lisps that are available.  Mention specific implementations you actually tested, SBCL or Clisp or whatever.  (It's true for both of these).  There's a reason these pages are called *implementation* contrasts.

Reply:
Whoa, you're right. It's undefined by the CL standard: http://clhs.lisp.se/Body/02_ce.htm. I was quite sure this was standard behavior. (`package::name` is the syntax for using symbols that may not be exported from the package; CL keywords are symbols in the KEYWORD package so that's why two colons are the same as one for keywords)